### PR TITLE
Added relay example.

### PR DESCRIPTION
**Pull request by mbardoeChoate**
I haven't added code to another person's repository before, so I am trying to start small. I saw the request for more examples and tried to pick the easiest thing I could. Probably not a super helpful example, but if I can do this well, I will try to pick off more important stuff later.

### CODE REVIEW

File: relay/robot.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    def teleopPeriodic(self):
+        """
+        The relay can be set several ways. It has two outputs which each can be set for either 0V or 12V.
+        This code uses buttons on a joystick to set each of the outputs.
+        """
+
+        forward = self.joystick.getRawButton(self.relayForwardButton)
+        reverse = self.joystick.getRawButton(self.relayReverseButton)
+
+        if forward and reverse:
```
Comment (virtuald) [inline relay/robot.py:37]: We try to keep the original comments found in the Java example, please add them:

* https://github.com/wpilibsuite/allwpilib/blob/96914143ba554cf2644229ddf54d1f4bead0a654/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/relay/Robot.java#L27
* https://github.com/wpilibsuite/allwpilib/blob/96914143ba554cf2644229ddf54d1f4bead0a654/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/relay/Robot.java#L34
* etc

File: relay/robot.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        # create joystick object
+        self.joystickChannel = 0  # usb number in DriverStation
+        self.joystick = wpilib.Joystick(self.joystickChannel)
+
+        # create variables to define the buttons
+        self.relayForwardButton = 1
+        self.relayReverseButton = 2
+
+    def teleopPeriodic(self):
+        """
```
Comment (virtuald) [inline relay/robot.py:29]: This comment isn't present in the original example, so let's omit it.

**Review by virtuald — CHANGES_REQUESTED** [review]
Mostly looks good, thanks! Just a few comment related adjustments, see https://github.com/robotpy/examples/blob/main/CONTRIBUTING.md#include-all-the-comments and the rest of that doc. 

Thanks!

**Comment by mbardoeChoate** [comment]
Tried to fix the comments. Thanks for the quick feedback.

File: relay/robot.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    output; pressing the button sets the output to 12V and releasing sets it to 0V.
+    """
+
+    def robotInit(self):
+        """Robot initialization function"""
+
+        self.relay = wpilib.Relay(0)
+
+        self.joystickChannel = 0
+        self.joystick = wpilib.Joystick(self.joystickChannel)
```
Comment (auscompgeek) [inline relay/robot.py:22]: The Java/C++ example doesn't define a separate variable for the joystick channel number; let's avoid introducing one here.
```suggestion
        self.joystick = wpilib.Joystick(0)
```

**Comment by virtuald** [comment]
Thanks!

**Pull request merged**